pkg/storage/export/export/export.go

```diff
@@ -511,7 +511,15 @@ func (ctrl *VMExportController) updateVMExport(vmExport *exportv1.VirtualMachine
 		return ctrl.handleSource(vmExport, service, ctrl.getPVCFromSourceVMSnapshot, ctrl.updateVMExporVMSnapshotStatus)
 	}
 	if ctrl.isSourceVM(&vmExport.Spec) {
-		return ctrl.handleSource(vmExport, service, ctrl.getPVCFromSourceVM, ctrl.updateVMExportVMStatus)
+		pvcTime, err := ctrl.handleSource(vmExport, service, ctrl.getPVCFromSourceVM, ctrl.updateVMExportPvcStatus)
+		if err != nil {
+			return pvcTime, err
+		}
+		vmTime, err := ctrl.handleSource(vmExport, service, ctrl.getPVCFromSourceVM, ctrl.updateVMExportVMStatus)
+		if err != nil {
+			return vmTime, err
+		}
+		return pvcTime + vmTime, nil
```

Comment:
Wouldn't we want to return `pvcTime + vmTime` here no matter what, or is there another reason for just returning `vmTime` here? Otherwise if there is no PVC status error but there is VM status error, the `pvcTime` is never returned or used; is this intentional? 

```suggestion
		return pvcTime + vmTime, err
```

Reply:
The commit message has been modified, thank you very much for reminding me of my mistake, the return time has been modified